package.json

```diff
@@ -61,22 +61,22 @@
     "node": ">=10"
   },
   "devDependencies": {
-    "codecov": "^3.6.5",
-    "eslint": "^7.3.1",
-    "eslint-config-airbnb-base": "^14.1.0",
+    "codecov": "^3.8.1",
+    "eslint": "^7.19.0",
+    "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^7.0.0",
-    "eslint-plugin-import": "^2.20.2",
-    "jest": "^26.0.1",
+    "eslint-plugin-import": "^2.22.1",
+    "jest": "^26.6.3",
     "jest-mock-now": "^1.3.0",
-    "jest-watch-typeahead": "^0.6.0"
+    "jest-watch-typeahead": "^0.6.1"
   },
   "dependencies": {
     "@solidity-parser/parser": "^0.11.0",
     "dir-to-object": "^2.0.0",
-    "emoji-regex": "^9.0.0",
+    "emoji-regex": "^9.2.1",
     "escape-string-regexp": "^4.0.0",
-    "prettier": "^2.0.5",
-    "semver": "^7.3.2",
+    "prettier": "^2.2.1",
+    "semver": "^7.3.4",
     "solidity-comments-extractor": "^0.0.4",
     "string-width": "^4.2.0"
```

Comment:
did you have to update all these deps yourself? have they been missed by dependabot? really?

Reply:
yeah dependabot does the heavy lifting for critical dependencies I believe. also I did not have any error pop up when I created the new tests and only started failing when I updated the dependancies. maybe the latest prettier did the trick.

Reply:
splendid, I'll merge and deploy thanks @Janther